content/strategy-goals/strategy/code-graph/code-intelligence/index.md

```diff
@@ -79,55 +79,54 @@ Recent key learnings:
 
 ## Strategy and Plans
 
-### FY22 Q4 goals
-
-These tie back to [FY22 Q4 Product/Engineering OKRs](../../../goals/2022_Q4.md#product-engineering)
+### Themes
```

Comment:
This is a lot of themes to all be working on at once. Are some more important than others? Do you really expect to work on all of them actively now?

If some are things we want to get to later, they would go in what we're not working on and why (with the reason being that some of the other things are more urgent/important).

Reply:
This made me realize I was leaving out significant items from the What's next section. Thanks for the thoughtful catch!